jcl/src/jdk.management/share/classes/com/ibm/lang/management/MemoryMXBean.java

```diff
@@ -274,4 +276,10 @@
      * @return number of active GC worker threads
      */
 	public int getCurrentGCThreads();
+	
+	/**
+	 * Returns information on the number and size of objects in the heap.
+	 * @return Array of information objects organized by class
+	 */
+	public HeapClassInformation[] getHeapClassStatistics();
```

Comment:
Ideally this should be implemented to be compatible with sun.tools.attach.HotSpotVirtualMachine.heapHisto(Object... var1) so any app using this will work with OpenJ9.
The acceptable options are "-live" or "-all"